packages/fxa-auth-server/lib/payments/subscription-reminders.ts

```diff
@@ -0,0 +1,216 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Logger } from 'mozlog';
+import Stripe from 'stripe';
+import { Container } from 'typedi';
```

Comment:
I’m not using `AppError` anywhere; in general unlike the PayPal processor script -- should I be? (Any thoughts on additional errors I should throw or log would be great.)

Reply:
hmmm I don't see the PayPal processor script using `AppError` either?  Anyways, the PayPal processor makes a lot of calls to both the PayPal NVP API and Stripe API, so a lot more can go wrong.

Reply:
Huh; you're right. Where was I seeing `AppError`? O_O Very true that the PayPal processor is making a lot more API calls to both Stripe and PayPal.